index.html

```diff
@@ -66,6 +70,11 @@
         </tbody>
       </table>
     </div>
+    <div id="footer">
+    {{ if .IsV6 }}
+      <a href="http://v4.ifconfig.co">IPv4 page</a>
```

Comment:
I'd rather have both links in the footer (http://v4.ifconfig.co and http://v6.ifconfig.co), without any logic.